app/assets/stylesheets/main.scss

```diff
@@ -27,6 +27,76 @@ main {
   }
 }
 
+#dashboard {
+  display: grid;
+  grid-template: 100px 150px 1fr 1fr 1fr 1fr 1fr 100px/  1fr 1fr 1fr 1fr 1fr 1fr;
+  background-color: lightgrey;
+
+  nav {
+    grid-area: 1 / 1 / 2 / 7;
+    display: grid;
+    grid-template: 1fr / 50px 1fr 1fr 1fr 50px;
+    background-color: darkgrey;
+    padding: 17px 0 0 0;
+
+    .username {
+      grid-area: 1 / 3 / 1 / 4;
+      color: white;
+      font-size: 45px;
+      font-weight: bold;
+      text-align: center;
+    }
+  }
+
+  footer {
+    grid-area: 8 / 1 / 9 / 7;
+    display: grid;
+    grid-template: 1fr / 75px 1fr 1fr 1fr 75px;
```

Comment:
I am really excited to use this in the project. Thanks for being daring enough to give it a shot!